lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -16,12 +16,13 @@ module Fastlane
 
   BUMP_PER_LABEL = {
     major: %w[pr:breaking].to_set,
-    minor: %w[pr:feat pr:minor].to_set,
-    patch: %w[pr:docs pr:fix pr:perf pr:dependencies pr:phc_dependencies pr:revenuecatui].to_set,
-    skip: %w[pr:build pr:ci pr:refactor pr:style pr:test pr:next_release].to_set
+    minor: %w[pr:feat pr:force_minor].to_set,
+    patch: %w[pr:fix pr:dependencies pr:phc_dependencies pr:force_patch].to_set,
+    skip: %w[pr:other pr:next_release].to_set
```

Comment:
We will need to update the `Dangerfile` with these new/old labels

Reply:
https://github.com/RevenueCat/Dangerfile/pull/10